schemas/context/pushdetail.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "meta:license": [
+        "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/pushdetail",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Push notification detail",
+    "type": "object",
+    "meta:extensible": true,
+    "description": "Push notification detail required to send mobile push notification to a specific mobile application on a given device.",
+    "definitions": {
+        "pushdetail": {
+            "properties": {
+                "xdm:token": {
+                    "title": "Push Token",
+                    "type": "string",
+                    "description": "Valid push token for the given push platform."
+                },
+                "xdm:platform": {
+                    "title": "Push Platform",
+                    "type": "string",
+                    "description": "Push provider, e.g. apns or fcm"
+                },
+                "xdm:blacklisted": {
+                    "title": "Blacklist Flag",
+                    "type": "boolean",
+                    "description": "Indicates whether the push token is blacklisted."
+                },
+                "xdm:appID": {
+                    "title": "Application Identifier",
+                    "type": "string",
+                    "description": "Unique Identifier for the mobile application."
+                },
+                "xdm:tokenID": {
```

Comment:
shouldn't be ID - since it's not the same type of ID as appID.   Could this be just `token`?

Reply:
Good point. I feel we should make it more enriched by saying "tokenSpecifications" or "tokenDetails"

Reply:
would "identity" work? I don't think this key is reserved and "identity" is the more accurate term here, this key holds the "active" identity on the app in the mobile phone when we collect the push token from the OS.

Reply:
I am OK with `xdm:identity` since that seems to be what it is now...